test/CoreTest.cpp

```diff
@@ -86,35 +84,23 @@ BOOST_AUTO_TEST_CASE(output_default_test)
 BOOST_AUTO_TEST_CASE(output_constructor_test)
 {
     using IE = IterationEncoding;
-    Series o1 = Series::create("./MyOutput_%T.dummy");
-
-    BOOST_TEST(o1.openPMD() == "1.0.1");
-    BOOST_TEST(o1.openPMDextension() == static_cast<uint32_t>(0));
-    BOOST_TEST(o1.basePath() == "/data/%T/");
-    BOOST_TEST(o1.meshesPath() == "meshes/");
-    BOOST_TEST(o1.particlesPath() == "particles/");
-    BOOST_TEST(o1.iterationEncoding() == IE::fileBased);
-    BOOST_TEST(o1.iterationFormat() == "MyOutput_%T");
-    BOOST_TEST(o1.iterations.size() == 0);
-    BOOST_TEST(o1.numAttributes() == 7); /* openPMD, openPMDextension, basePath, meshesPath, particlesPath, iterationEncoding, iterationFormat */
-    BOOST_TEST(o1.name() == "MyOutput_%T");
-
-    o1.iterations[0];
-
-    Series o2 = Series::create("./MyCustomOutput.dummy");
-
-    o2.setMeshesPath("customMeshesPath").setParticlesPath("customParticlesPath");
-
-    BOOST_TEST(o2.openPMD() == "1.0.1");
-    BOOST_TEST(o2.openPMDextension() == static_cast<uint32_t>(0));
-    BOOST_TEST(o2.basePath() == "/data/%T/");
-    BOOST_TEST(o2.meshesPath() == "customMeshesPath/");
-    BOOST_TEST(o2.particlesPath() == "customParticlesPath/");
-    BOOST_TEST(o2.iterationEncoding() == IE::groupBased);
-    BOOST_TEST(o2.iterationFormat() == "/data/%T/");
-    BOOST_TEST(o2.iterations.size() == 0);
-    BOOST_TEST(o2.numAttributes() == 7); /* openPMD, openPMDextension, basePath, meshesPath, particlesPath, iterationEncoding, iterationFormat */
-    BOOST_TEST(o2.name() == "MyCustomOutput");
+    Series o = Series::create("./MyCustomOutput.dummy");
+
+    o.setMeshesPath("customMeshesPath").setParticlesPath("customParticlesPath");
+
+    o.iterations[1].meshes["foo"];
+    o.iterations[1].particles["bar"];
+
+    BOOST_TEST(o.openPMD() == "1.1.0");
+    BOOST_TEST(o.openPMDextension() == static_cast<uint32_t>(0));
+    BOOST_TEST(o.basePath() == "/data/%T/");
+    BOOST_TEST(o.meshesPath() == "customMeshesPath/");
+    BOOST_TEST(o.particlesPath() == "customParticlesPath/");
+    BOOST_TEST(o.iterationEncoding() == IE::groupBased);
+    BOOST_TEST(o.iterationFormat() == "/data/%T/");
+    BOOST_TEST(o.iterations.size() == 1);
+    BOOST_TEST(o.numAttributes() == 7); /* openPMD, openPMDextension, basePath, meshesPath, particlesPath, iterationEncoding, iterationFormat */
```

Comment:
it's always allowed to have more attributes than openPMD defines, so this is a sub-ideal test. maybe make it `>=7`?

Reply:
I don't see how this is sub-ideal. ```numAttributes``` counts the number of currently set Attributes. As this test happens in a very conttrolled environment, we should check precisely whether the defaults reflect the standard in that case.

Reply:
oh wait, it's a write test not a read test. you are right!